src/modules/import/file.js

```diff
@@ -10,7 +10,7 @@ goog.provide('ngeo.File');
  * @ngInject
  */
 ngeo.File = function($q, $http, $window, gettext) {
-  var fileReader, canceler;
+  let fileReader, canceler;
```

Comment:
The import module may be used independently of the rest of the code, with a different build system.
I see that babel that you used is able to convert a whole directory to ES5. Do you think it is OK?

https://babeljs.io/docs/usage/cli/

Compile the entire src directory and output it to the lib directory. You may use --out-dir or -d. This doesn’t overwrite any other files or directories in lib.
```
babel src/modules/import --out-dir es5_import
```
Compile the entire src directory and output it to the one concatenated file.
```
babel src/modules/import --out-file es5_import_one_file.js
```